### DISCUSSION TITLE
Archive on phone not in sync with archive in web app

### DISCUSSION THREAD

**Discussion by n4tur4lb0rnch1ll4** [immich-app/immich#7897] [Q&A] [answered]
Hello dear immich community,

as the title says - the archives aren't in sync and I would like to have one cloud saved archive that looks the same on either device, how can that be achieved?

Thanks for any hints upfront!

Flo

**Comment by schuhbacca** [comment, marked as answer]
Do you have the latest app and sever? From the latest release notes.
![image](https://github.com/immich-app/immich/assets/5739463/b2fd4e3a-c19d-417d-a6ee-987f35b27c17)

**Reply by alextran1502** [reply]
@schuhbacca you are too fast!

**Reply by n4tur4lb0rnch1ll4** [reply]
Thank you so much!
